README.md

```diff
@@ -46,31 +58,18 @@ Open the file `carto-values.yaml` that you have received and configure the neede
 - Configure [external DBs](#external-databases)
 - Configure external buckets
 
-4. Deploy CARTO
-
-```bash
-helm install carto-selfhosted-v1 carto-selfhosted-charts/carto -f carto-values.yaml -f carto-secrets.yaml
-```
-
-5. Configure your DNS to point to the deployment:
-
-  In GKE:
-
-  Create an A record pointing to the IP:
-
-  ```bash
-  kubectl get svc carto-selfhosted-v1-router -o jsonpath='{.status.loadBalancer.ingress.*.ip}'
-  ```
-  
-  In EKS:
-
-  Create a CNAME record poiting to:
-
+5. Install your deployment:
   ```bash
-  kubectl get svc carto-selfhosted-v1-router -o jsonpath='{.status.loadBalancer.ingress.*.hostname}'
+  helm install \
+    <your_own_installation_name|carto> \
+    carto-selfhosted/carto \
+    --namespace <your_namespace> \
+    -f carto-values.yaml \
+    -f carto-secrets.yaml \
+    <other_custom_files>
```

Comment:
do we really need to add the `<other_custom_files>`??

Reply:
yeah, is the way to add all the configurations like the new `Made it public` and more to come !